tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -831,10 +831,10 @@ describe('runs_table', () => {
           })
         );
 
-        const dialogInputDebugElement: DebugElement = new DebugElement(
+        const dialogInputDebugElement = new DebugElement(
           overlayContainer
             .getContainerElement()
-            .querySelector('mat-dialog-container input')
+            .querySelector('mat-dialog-container input')!
```

Comment:
The `:DebugElement` removal was due to the lint `redundant type annotation` for the `new` statement.  The error I ran into was saying the element passed to the DebugElement constructor for  `new DebugElement(element: Element)` has a type mismatch, without `!` it will be `Element | null` assigned to `Element`. See the Builder error in cl/454682502.

Although yesterday we talked about how we couldn't run tests for these internally anyway, so the errors would only show up if we change `runs_table_test.ts` and try to import it, otherwise it wouldn't really affect anything.